test/orchard/info_test.clj

```diff
@@ -115,6 +115,23 @@
           (is (= expected (select-keys i [:ns :name :arglists])))
           (is (str/includes? (:file i) "test_ns_dep")))))))
 
+(deftest info-bug-53-test
+  (testing "test for bug #53"
+    (let [expected '{:ns clojure.string
+                     :name replace
+                     :arglists ([s match replacement])}]
+      (testing "- :cljs"
+        (let [i (info/info 'orchard.test-ns 'clojure.string/replace *cljs-params*)]
+          (is (= expected (-> (info/info 'orchard.test-ns 'clojure.string/replace *cljs-params*)
+                              (select-keys [:ns :name :arglists]))))
+          (is (str/starts-with? (:doc i) "Replaces all instance of match with replacement in s."))))
```

Comment:
I think checking that doc makes the test a bit brittle, as the docstring might be improved in future version of Clojure. Not a huge concern but such checks caused issues in the past.

Reply:
Ok I will take it away